galaxy/api/v2/tests/test_collection_view.py

```diff
@@ -0,0 +1,32 @@
+from django.test import TestCase
+from rest_framework.test import APIClient
+
+from galaxy.main import models
+
+
+class CollectionViewTests(TestCase):
+
+    def setUp(self):
+        self.namespace = models.Namespace.objects.create(
+            name='mynamespace',
+        )
```

Comment:
Probably better to cleanup created namespaces after test is finished.

`self.addCleanup(self.namespace.delete)`

Reply:
As stated, this is not necessary. All fixtures are already cleaned up between tests.

Reply:
Ok